### ISSUE TITLE
How to convert to .exe

### ISSUE THREAD

**Issue by CoderZ90** [Rachit333/hand_gesture_mouse_control#3]
How to convert to .exe

**Comment by CoderZ90** [comment]
Please help i am not able to convert it ?

**Comment by Rachit333** [comment]
Sorry for late reply. Please refer to this link to solve the issue: https://youtu.be/Y0HN9tdLuJo
If you have any issues let me know :)

**Comment by Rachit333** [comment]
Or refer to this link.

**Comment by Rachit333** [comment]
Or refer to https://youtu.be/UZX5kH72Yx4 << link.

**Comment by Rachit333** [comment]
I tried converting .py to .exe through following above links and I was able to convert to .exe but it just opens up and closes instantly giving this error:
![error](https://user-images.githubusercontent.com/64753500/123500518-43eef080-d65c-11eb-8bb5-e2ccc0ba8384.png)
So basically _**pynput**_ and  _**mediapipe**_ is giving us that import error.

**Comment by Rachit333** [comment]
For **_pynput_** if we put `--hidden-import "pynput.keyboard._win32" --hidden-import "pynput.mouse._win32"` after `pyinstaller --onefile HGMC.py` it solves half the issue.

**Comment by Rachit333** [comment]
But the other half....idk :(

**Comment by Rachit333** [comment]
Here's the error photo if you want to know..
![error1](https://user-images.githubusercontent.com/64753500/123501301-aa2a4200-d661-11eb-9980-ec653817def2.png)

**Comment by CoderZ90** [comment]
Yes same error i am getting

**Comment by Rachit333** [comment]
> Yes same error i am getting

Hey! Ik I am answering very late, but I think i found this solution. https://github.com/google/mediapipe/issues/2162#issuecomment-974658826
This might help. Although i havent tested it myself for now.
I just happened to make an .exe from some other mediapipe project and ran into the same problem. So, therefore I searched github for the issue and found the solution, hope this helps...now. 
sorry for not answering quicker.😊
